harvard-dcu.csl

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Harvard (Dublin City University)</title>
+    <title-short>Harvard (DCU)</title-short>
+    <id>http://www.zotero.org/styles/harvard-dcu</id>
+    <link rel="self" href="http://www.zotero.org/styles/harvard-dcu"/>
+    <link href="http://www.zotero.org/styles/harvard" rel="template"/>
+    <link href="https://www101.dcu.ie/library/Citing&amp;ReferencingGuide/player.html" rel="documentation"/>
+    <author>
+      <name>Jack Hyland</name>
+      <email>jack.hyland@dcu.ie</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <published>2016-12-12T14:36:00+00:00</published>
+    <updated>2016-12-12T14:36:07+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with=". "/>
+      <label form="short" text-case="lowercase" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="0" et-al-use-first="4" et-al-use-last="true" initialize-with="." name-as-sort-order="all"/>
```

Comment:
Hi @rmzelle - I've made those three changes:
-template linking to a diff. style
-deleted the anon module
-sorted the et al issue (hopefully)